main.py

```diff
@@ -3993,6 +4001,9 @@ def GTM_CellChange_fun(self):
         if value == "":
             if (row in Data["General"]["DataSection"][str(column)]):
                 Data["General"]["DataSection"][str(column)].pop(row)
+                item.setBackground(Qt.red)
+                list_item = self.add_delete_error_in_TM(key, mode = "add")     
+                Data["error_in_TM"][key] = list_item
```

Comment:
**Why** we need these lines of codes and **what** they do?
It seems to changes cell color to red at GTM every time a cell gets cleared, its right?

Reply:
because we shouldn't have demands without source or destination...